pom.xml

```diff
@@ -58,6 +58,10 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <scala.version>${kafka.scala.version}.8</scala.version>
         <scalatest.version>3.0.5</scalatest.version>
+        <docker.os_type>deb9</docker.os_type>
```

Comment:
do we want to build for deb9 as default?

Reply:
The default for this image should be deb9 because that is what this image has always used. this image was always based on cp-base-new which is a Debian 9 image.